ts/sagas/startup/loadServiceRequestHandler.ts

```diff
@@ -28,6 +39,28 @@ export function* loadServiceRequestHandler(
     }
 
     if (response.value.status === 200) {
+      // If the organization fiscal code is associated to different organizazion names,
+      // it is considered valid the one declared for a visible service
+      const visibleServices: VisibleServicesState = yield select(
+        visibleServicesSelector
+      );
+      const service = pot.some(response.value.value);
+      const isVisible = isVisibleService(visibleServices, service) || false;
+
+      const organizations: OrganizationNamesByFiscalCodeState = yield select(
+        organizationNamesByFiscalCodeSelector
+      );
+      const fc = service.value.organization_fiscal_code;
+
+      if (organizations) {
```

Comment:
```suggestion
      if (organizations) {
        // If the organization fiscal code is associated to different organizazion names,
        // it is considered valid the one declared for a visible service
        const visibleServices: VisibleServicesState = yield select(
          visibleServicesSelector
        );
        const service = pot.some(response.value.value);
        const isVisible = isVisibleService(visibleServices, service) || false;
        const fc = service.value.organization_fiscal_code;
        const organization = organizations[fc];
        // If the organization has been previously saved in the organization entity,
        // the organization name  is updated only if the related service is visible
        if (!organization || (organization && isVisible)) {
          yield put(updateOrganizations(response.value.value));
        }
      }
```
some values are needed only if the the block `if (organizations)` is executed

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1328/commits/91b927c1d9896ad446400c5e1e43c042f0ca8877